test: manual test fixes and cleanup

NoelStephensUnity opened the pull request:
This is just a quick PR to cleanup and fix the SceneTransitioning and SceneTransitioningAdditive manual tests.

## Testing and Documentation
* No tests have been added.
* No documentation changes or additions were necessary.

NoelStephensUnity committed:
fix

Somehow SceneTransitioningTest's NetworkManager was still configured for UNet but set to use the UnityTransport.  This fixes the issue and increases the UnityTransport maximum buffer size to 524288.

NoelStephensUnity committed:
chore

rename the two primary scenes that are loaded when switching scenes to:
SceneTransitioningAdditive: the primary scene with the NetworkManager in it.
SceneTransitioningAdditiveSwitch: the second scene that is loaded when switching scenes (i.e. loaded in single mode).

NoelStephensUnity committed:
update

This updates the SceneTransitionAdditive manual test to use the UnityTransport by default.

jeffreyrainy commented:
It's a bit annoying that the changes are all .unity with the actual meaning/intent hidden away. But as long as CI passes and this makes the scene work. 👍

NoelStephensUnity committed:
Merge branch 'develop' into test/manual-test-fixes-and-cleanup

NoelStephensUnity committed:
test fix UnityTransporrt

This fixes the issue with UnityTransport and the Server Listen Address being String.Empty. MTT-2296

NoelStephensUnity committed:
Merge branch 'develop' into test/manual-test-fixes-and-cleanup

NoelStephensUnity committed:
Revert "test fix UnityTransporrt"

This reverts commit 1092639791dbdb787d868dfe96477a318b536bb6.

NoelStephensUnity committed:
fix

Switching to UnityTransport for the prefab pool examples.  This fixes an issue with a native collection not being released.

NoelStephensUnity committed:
Merge branch 'develop' into test/manual-test-fixes-and-cleanup

NoelStephensUnity committed:
Merge branch 'develop' into test/manual-test-fixes-and-cleanup

NoelStephensUnity committed:
Merge branch 'develop' into test/manual-test-fixes-and-cleanup

NoelStephensUnity commented:
No longer needed these specific changes, the scenetransitioningadditive is going to be updated in PR-1646
